rusoto/credential/src/lib.rs

```diff
@@ -617,7 +617,7 @@ mod tests {
         );
     }
 
-    #[test]
+    #[cfg(test)]
```

Comment:
This appears to work, as I could get `cargo test` to fail by making a little switch to the assertion helper:

```
-    !debug.contains(SECRET) && debug.contains("**********")
+    !debug.contains(SECRET) && debug.contains("🤘")```